packages/extension/src/content/index.tsx

```diff
@@ -1,25 +1,29 @@
 import { browser } from 'webextension-polyfill-ts';
 
-// Inject app div
-const appContainer = document.createElement('daily-companion-app');
-document.body.appendChild(appContainer);
+const hasRendered = !!document.querySelector('daily-companion-app');
```

Comment:
for some reason I felt like `isRendered` made more sense looking at the rest of our code-base (but it's an optional change)

Reply:
Totally agree. Let me update it quickly! 🚢